service-api/docker/web/etc/confd/templates/nginx.conf.dev

```diff
@@ -0,0 +1,115 @@
+log_format tracing escape=json '{'
```

Comment:
Whereabouts is this used in the build?

Reply:
This is used to get the service-api to log what json was sent from the front-end to the api.  It can be copied over to nginx.conf to achieve this.  I debated whether to have this file checked into git or not , as how to do this is also described in logging.md. However, since I have the file locally, seems a shame not to check it in as an example.  What's your opinion?  I was somewhat undecided....

Reply:
It's OK, but worth a comment explaining why it's here and pointing to logging.md?

Reply:
Agreed. Now done :)